### DISCUSSION TITLE
Building Go v1 apps without go mod

### DISCUSSION THREAD

**Discussion by suntong** [goreleaser/goreleaser#2148] [Q&A] [answered]
I changed my repo
https://github.com/suntong/example/commits/master

to without go mod, 
https://github.com/suntong/example/commit/2a57053ad1d29aaacc64bad7d7a33bf9db2e140e

and the build failed

```
🏃 Running GoReleaser...
17
/opt/hostedtoolcache/goreleaser-action/0.161.1/x64/goreleaser release --rm-dist
18
   • releasing...     
19
   • loading config file       file=.goreleaser.yaml
20
   • loading go mod information
21
   ⨯ release failed after 0.01s error=failed to get module path: exit status 1: go list -m: not using modules
```

Is it only working for projects with go mod? 
I still have lots of v1 projects with them. 

Any way to get them supported as well? Thx

**Comment by caarlos0** [comment, marked as answer]
hmm, seems like a bug actually, can you open an issue instead? Thanks!

**Reply by suntong** [reply]
Sure, #2149.

**Reply by caarlos0** [reply]
thanks!
